.editorconfig

```diff
@@ -0,0 +1,36 @@
+# This is the top-most EditorConfig file.
+root = true
+
+# For all files.
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = false
+
+# Shell scripts
+[*.sh]
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
```

Comment:
Couldn't this be moved to `[*]` for all files?

Reply:
It can. It was done with the purpose to make the first version very easy to review.

Reply:
Fixed